### PR TITLE
Describe status of the specifications

### LINKED ISSUE #47: Add Status Of This Document
Communicate about the status of the specs: there is no implementation, the design goals have been to capture the use cases and interaction patterns needed to meet them. We assume further spec versions would be an outcome of an implementation phase.

### PR DESCRIPTION
**Pull request by no-reply**
Closes #47.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,12 @@ application. They each provide an HTTP-based approach for authentication, commun
...
 * [OTM Audit Events](audit-appendix.html) - A listing of preservation events that occur as part the workflow that may be captured by the DDP and made available through the Bridge
 
+### Status of the Specifications
+These specifications were created as part of the One to Many grant, funded by the Andrew W. Mellon Foundation. The goals
+and scope of this grant are described in [One to Many Project Overview and Goals](https://wiki.lyrasis.org/display/OTM/Project+Overview+and+Goals)</a>.
+
+The specifications aim provide the structure and interaction patterns necessary to meet the [One to Many User Stories](https://wiki.lyrasis.org/display/OTM/User+Stories). The team sought to define the APIs with enough specificity to allow future implementation efforts to succeed in delivering systems that satisfy those user stories, while leaving flexibility for implementers to adjust to the needs of the development phase. Future versions of these specifications are expected as an outcome of any effort to implement them.
```
Comment (SibylSchaefer) [inline README.md]: Change to "These specifications aim to"

File: README.md [hunk 2]
```diff
@@ -29,6 +29,12 @@ application. They each provide an HTTP-based approach for authentication, commun
 * [OTM Bridge API Specification](otm-bridge.html) - The Bridge provides a consistent interface for systems (including repositories) to integrate with DDP platforms; the Bridge also serves as a staging point for content being transferred into or out of a DDP
 * [OTM Audit Events](audit-appendix.html) - A listing of preservation events that occur as part the workflow that may be captured by the DDP and made available through the Bridge
 
+### Status of the Specifications
+These specifications were created as part of the One to Many grant, funded by the Andrew W. Mellon Foundation. The goals
```
Comment (no-reply) [inline README.md]:
```suggestion
These specifications were created as part of the <a href="https://mellon.org/grants/grants-database/grants/university-of-california-at-san-diego/1805-05809/">One to Many grant</a>, 
funded by the Andrew W. Mellon Foundation. The goals 
```

File: README.md [hunk 3]
```diff
@@ -29,6 +29,13 @@ application. They each provide an HTTP-based approach for authentication, commun
 * [OTM Bridge API Specification](otm-bridge.html) - The Bridge provides a consistent interface for systems (including repositories) to integrate with DDP platforms; the Bridge also serves as a staging point for content being transferred into or out of a DDP
 * [OTM Audit Events](audit-appendix.html) - A listing of preservation events that occur as part the workflow that may be captured by the DDP and made available through the Bridge
 
+### Status of the Specifications
+These specifications were created as part of the <a href="https://mellon.org/grants/grants-database/grants/university-of-california-at-san-diego/1805-05809/">One to Many grant</a>,
```
Comment (no-reply) [inline README.md]:
```suggestion
These specifications were created as part of the [One to Many grant](https://mellon.org/grants/grants-database/grants/university-of-california-at-san-diego/1805-05809/)</a>, 
```

File: README.md [hunk 4]
```diff
@@ -29,6 +29,13 @@ application. They each provide an HTTP-based approach for authentication, commun
 * [OTM Bridge API Specification](otm-bridge.html) - The Bridge provides a consistent interface for systems (including repositories) to integrate with DDP platforms; the Bridge also serves as a staging point for content being transferred into or out of a DDP
 * [OTM Audit Events](audit-appendix.html) - A listing of preservation events that occur as part the workflow that may be captured by the DDP and made available through the Bridge
 
+### Status of the Specifications
+These specifications were created as part of the [One to Many grant](https://mellon.org/grants/grants-database/grants/university-of-california-at-san-diego/1805-05809/)</a>,
```
Comment (no-reply) [inline README.md]:
```suggestion
These specifications were created as part of the [One to Many grant](https://mellon.org/grants/grants-database/grants/university-of-california-at-san-diego/1805-05809/), 
```

File: README.md [hunk 5]
```diff
@@ -29,6 +29,13 @@ application. They each provide an HTTP-based approach for authentication, commun
 * [OTM Bridge API Specification](otm-bridge.html) - The Bridge provides a consistent interface for systems (including repositories) to integrate with DDP platforms; the Bridge also serves as a staging point for content being transferred into or out of a DDP
 * [OTM Audit Events](audit-appendix.html) - A listing of preservation events that occur as part the workflow that may be captured by the DDP and made available through the Bridge
 
+### Status of the Specifications
+These specifications were created as part of the [One to Many grant](https://mellon.org/grants/grants-database/grants/university-of-california-at-san-diego/1805-05809/), 
+funded by the Andrew W. Mellon Foundation. The goals 
+and scope of this grant are described in [One to Many Project Overview and Goals](https://wiki.lyrasis.org/display/OTM/Project+Overview+and+Goals)</a>.
```
Comment (no-reply) [inline README.md]:
```suggestion
and scope of this grant are described in [One to Many Project Overview and Goals](https://wiki.lyrasis.org/display/OTM/Project+Overview+and+Goals).
```

File: README.md [hunk 6]
```diff
@@ -29,6 +29,13 @@ application. They each provide an HTTP-based approach for authentication, commun
...
 * [OTM Audit Events](audit-appendix.html) - A listing of preservation events that occur as part the workflow that may be captured by the DDP and made available through the Bridge
 
+### Status of the Specifications
+These specifications were created as part of the [One to Many grant](https://mellon.org/grants/grants-database/grants/university-of-california-at-san-diego/1805-05809/), 
+funded by the Andrew W. Mellon Foundation. The goals 
+and scope of this grant are described in [One to Many Project Overview and Goals](https://wiki.lyrasis.org/display/OTM/Project+Overview+and+Goals).
+
+The specifications aim provide the structure and interaction patterns necessary to meet the [One to Many User Stories](https://wiki.lyrasis.org/display/OTM/User+Stories). The team sought to define the APIs with enough specificity to allow future implementation efforts to succeed in delivering systems that satisfy those user stories, while leaving flexibility for implementers to adjust to the needs of the development phase. Future versions of these specifications are expected as an outcome of any effort to implement them.
```
Comment (no-reply) [inline README.md]:
```suggestion
These specifications aim to provide the structure and interaction patterns necessary to meet the [One to Many User Stories](https://wiki.lyrasis.org/display/OTM/User+Stories). The team sought to define the APIs with enough specificity to allow future implementation efforts to succeed in delivering systems that satisfy those user stories, while leaving flexibility for implementers to adjust to the needs of the development phase. Future versions of these specifications are expected as an outcome of any effort to implement them.
```

**Pull request merged**